### ISSUE TITLE
Feature Request: Export/Import Mission State Between Characters.

### ISSUE THREAD

**Issue by Xen0byte** [apocalyptech/bl3-cli-saveedit#20]
Completing the same missions multiple times (once per character) is quite laborious. It would be nice to be able to have a "main" from which to copy the mission state to the other characters.

**Comment by apocalyptech** [comment]
Yeah, I'd actually started on an implementation of this but quickly got into the weeds with how to properly deal with mission-related challenges and the like, and it became too much of a datamining exercise for me to want to finish it up.  I've got a nascent function available in the backend library for it, though I make no promises as to if it actually works or not:

https://github.com/apocalyptech/bl3-cli-saveedit/blob/master/README-saves.md#copy-nvhm-state-to-tvhm

And an "unpackaged" CLI interface to it, over here:

https://github.com/apocalyptech/bl3-cli-saveedit/blob/master/bl3save/cli_copy_pt.py

I'm afraid that I just don't really care enough to take it any further on my own -- If someone was willing to put in the effort to fix it up and demonstrate that it Does The Right Thing and doesn't result in any janky weird saves, I'd be happy to get a PR merged in.  Failing that, of course, folks can play around with editing the JSON exports to their hearts' content, so that'd be a way of doing it manually, at least.

**Comment by Xen0byte** [comment]
OK, dumping the save file to JSON, manually editing that, and then wrapping it back to *.sav format seems to work fine. I just managed to restart a bugged mission today, so thanks for pointing me in the right direction. I'll look into migrating the mission pool progress from one save file to another sometime within the next few days. Off the top of my head, I don't see how that would have any side effects, but I'll come back with a comment on my findings once I finish.
